src/components/Projects/CreateProject/Files/Manifest.ts

```diff
@@ -29,8 +29,10 @@ export class CreateManifest extends CreateFile {
 					name: 'pack.name',
 					description: 'pack.description',
 					min_engine_version:
-						this.type === 'data'
-							? [1, 13, 0]
+						this.type === 'data' || 'resources'
+							? createOptions.targetVersion
```

Comment:
Are you sure that this will work? I believe we rolled back this twice now (v1 and v2 yesterday) because we've got reports that it won't work with anything other than "1.13.0"

Reply:
I just tested this again to be sure. No problems, the packs show up fine. When I am developing add-ons, I always set the min_engine_version to the latest version. Never had problems with that. I think the problem with v2 was that it wasn't adding the min_engine_version to the Rp manifest, which caused the pack not showing up inside Minecraft?
![image](https://user-images.githubusercontent.com/7486865/107880551-ceba2480-6edf-11eb-9f02-3e444c05bc05.png)
![image](https://user-images.githubusercontent.com/7486865/107880543-bfd37200-6edf-11eb-8d61-08a72dab16f9.png)

Reply:
Alright, perfect 👏 
Thanks!